<a href="https://colab.research.google.com/github/PosgradoMNA/actividades-del-projecto-equipo-48/blob/main/Actividad_2_de_proyecto_final.ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

**Nombre  y datos de los integrantes del equipo:**
* Roberto Romero Vielma | Matrícula: A00822314 | Github ID: #104024024
* Franciso Medellín Zertuche | Matrícula: A01794044 | Github ID: #63127790


#**Conjunto de datos a utilizar en el proyecto**
La problemática que se quiere conocer es de tipo de social, en este caso referente al crimen en México, a continuación se muestra la base de datos:

In [ ]:
import pandas as pd
file="https://raw.githubusercontent.com/PosgradoMNA/actividades-del-projecto-equipo-48/main/IDEFC_NM_may22.csv"
df=pd.read_csv(file, encoding='latin-1')

In [ ]:
df.head()

Año  Clave_Ent         Entidad            Bien jurídico afectado  \
0  2015          1  Aguascalientes  La vida y la Integridad corporal   
1  2015          1  Aguascalientes  La vida y la Integridad corporal   
2  2015          1  Aguascalientes  La vida y la Integridad corporal   
3  2015          1  Aguascalientes  La vida y la Integridad corporal   
4  2015          1  Aguascalientes  La vida y la Integridad corporal   

  Tipo de delito  Subtipo de delito          Modalidad  Enero  Febrero  Marzo  \
0      Homicidio   Homicidio doloso  Con arma de fuego      3        0      2   
1      Homicidio   Homicidio doloso    Con arma blanca      1        1      0   
2      Homicidio   Homicidio doloso  Con otro elemento      0        0      2   
3      Homicidio   Homicidio doloso    No especificado      2        0      0   
4      Homicidio  Homicidio culposo  Con arma de fuego      0        0      0   

   Abril  Mayo  Junio  Julio  Agosto  Septiembre  Octubre  Noviembre  \
0      1     1    1.0    2.0     1.0         2.0      2.0        2.0   
1      0     0    1.0    0.0     1.0         0.0      0.0        0.0   
2      2     3    2.0    0.0     1.0         2.0      0.0        0.0   
3      1     0    0.0    0.0     0.0         0.0      0.0        0.0   
4      0     1    0.0    0.0     0.0         0.0      0.0        0.0   

   Diciembre  
0        1.0  
1        1.0  
2        0.0  
3        0.0  
4        0.0

**Cantidad de filas presentes en el dataset:**

In [ ]:
len(df)

25088

Se tiene un total de 25,088 filas.

#**Nombre que caracteriza el conjunto de datos**
El nombre del dataset es **Crime in Mexico Data**.

#**Organización que lo generó**
La organización que generó el conjunto de datos es el **Secretariado Ejecutivo del Sistema Nacional de Seguridad Pública (SESNSP)** del Gobierno de México.

# **Incluya un enlace de Internet donde se puedan descargar el base de datos en línea**
Enlace al dataset: https://www.kaggle.com/datasets/linigoga/crime-in-mexico-data?resource=download

# **Describe el conjunto de datos. ¿Qué datos contiene?**
El dataset contiene datos como el año en el que ocurrieron los incidentes delictivos, la entidad federativa donde sucedió, así como la clave de la entidad, también se detalla el bien jurídico afectado (puede ser la vida, patrimonio, familia o sociedad), además del tipo (homicidio por ejemplo) y subtipo (homicidio doloso o culposo) de delito cometido, así como la modalidad (con arma de fuego o con arma blanca), por último vienen varias columnas donde se detalla el mes en el que ocurrieron los incidentes.

# **¿Cuáles son algunas de las preguntas de investigación que podría responder con estos datos? Enumere al menos OCHO (no responda las preguntas)**

*	¿Qué mes es el más propenso a tener más homicidios?
*	¿Cuál es el estado con mayor tasa de homicidios?
*	¿Qué mes es el más propenso a tener más actos delictivos?
*	¿Cuál es el estado con mayor tasa de delitos?
*	¿Existen relaciones entre los diferentes tipos de delitos?
*	¿Cuál es el año con más actos delictivos?
*	¿Cuál es la modalidad más común de homicidios?
*	¿Cuál es el tipo de modalidad delictiva más frecuente por estado?
*	¿Qué estado presenta la mayor tasa de secuestros?
*	¿Ha habido alguna reducción de delitos en alguna entidad federativa?
*	¿Cuál es el tipo de delito que presenta una mayor reducción de casos?
 

# **¿Qué actor (empresa, sector de la sociedad, organismo descentralizado, gobierno, etc.) estaría interesado en las preguntas mencionadas que identificó?**

- Gobiernos Estatales.
- Gobierno Federal.
- Empresas de seguridad y vigilancia.
- Empresas de seguridad privada con armas
- Empresas de transporte de valores.
- ONGs relacionadas al desarrollo social.
- Aseguradoras.
- Grupos financieros y de inversión privada.
